### ISSUE TITLE
sdist is missing `tox.ini`

### ISSUE THREAD

**Issue by mtelka** [aio-libs/aiosignal#651]
The sdist package at PyPI is missing the `tox.ini` file.  Please add the missing file to sdist to make downstream testing easier.  Thank you.

**Comment by Dreamsorcerer** [comment]
I don't think we use tox anymore. It's all handled through Github actions. The file can probably be deleted.

**Comment by mtelka** [comment]
The `tox.ini` is the preferred way to run tests while packaging Python projects for OpenIndiana.  It is the only method that currently works fully automatic.  All other methods needs some extra manual work.  So if possible (and if the `tox.ini` is working) I vote for keeping it.

**Comment by webknjaz** [comment]
Yeah, I'd like to have it working, but it's currently unmaintained. I'm yet to fix the way of using it in the org but will probably start the experiments in propcache.

**Comment by webknjaz** [comment]
Solved.
